egui/src/ui.rs

```diff
@@ -783,7 +783,7 @@ impl Ui {
         &self.placer
     }
 
-    pub(crate) fn cursor(&self) -> Rect {
+    pub fn cursor(&self) -> Rect {
```

Comment:
If you make this public, you should add a documentation for it (i.e. or figure out its meaning itself, or ask @emilk)

Reply:
I agree. You can find the documentation for it in `Region::cursor`. Note that `cursor` is a complicated thing, and also that it may change in future egui versions if/when the layout engine is rewritten (again). This is why it hasn't been exposed (yet).

Reply:
I copied the docs from `Region::cursor`.